.github/CODEOWNERS

```diff
@@ -1,16 +1,20 @@
 # Documentation/Infrastructure
-/docs                                       @JustinyAhin @felixarntz
 /admin                                      @JustinyAhin @felixarntz
 /tests/admin                                @JustinyAhin @felixarntz
 /bin                                        @JustinyAhin @felixarntz
 /.github                                    @JustinyAhin @felixarntz
 
+# Focus: Database
+/modules/database
+/tests/modules/database
+/tests/testdata/modules/database
```

Comment:
Should @OllieJones be added here now? :)

Reply:
Sorry, missed your Slack mention. Updated!